.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "7"
+dist: trusty
+sudo: required
+group: edge
+script:
+  - npm run test-security && npm run test-index
```

Comment:
any reason not to also run `npm run test-integration`?

Reply:
No. Other than it's not fixed yet. :)

Reply:
@evq fixed this in #56, so we can add it as well

Reply:
Awesome! Done.